packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -970,10 +1005,9 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					editor.mark('rotate-cw')
 					const offset = editor.getSelectionRotation() % (HALF_PI / 2)
 					const dontUseOffset = approximately(offset, 0) || approximately(offset, HALF_PI / 2)
-					editor.rotateShapesBy(
-						editor.getSelectedShapeIds(),
-						HALF_PI / 2 - (dontUseOffset ? 0 : offset)
-					)
+					const selectedShapeIds = editor.getSelectedShapeIds()
+					editor.rotateShapesBy(selectedShapeIds, HALF_PI / 2 - (dontUseOffset ? 0 : offset))
+					kickoutOccludedShapes(editor, selectedShapeIds)
```

Comment:
it feels like we're going to forget to do `kickoutOccludedShapes` in places - is there a way to do this in a more unified place for general transform operations?

Reply:
discussed offline - I could see us moving this function into the editor methods themselves, but I feel more comfortable tying 'kicking out' to user actions, rather than agnostic editor methods. But it does feel fragile to have to remember to use the `kickoutOccludedShapes` in all these places. I'm ok with having it like this, but also happy to move it elsewhere 🤷